discord-bot/src/commands/animatedEmoji/index.ts

```diff
@@ -13,12 +13,13 @@ const animatedEmoji = async (originalMessage: Message) => {
   const emojiRegex = new RegExp('(:.+:)+', 'gi');
   const hasEmoji = content.match(emojiRegex);
   if (!hasEmoji) return; // return if no emoji found
-  const textChannel = channel as TextChannel;
-  let webhook = await fetchWebhook(textChannel);
-  if (!webhook) {
-    webhook = await createWebhook(textChannel); // create webhook if not found
-  }
-  if (!webhook) return; // return if can't find or create webhook
+
+  const currentTextChannel = channel as TextChannel;
+  const webhook = await fetchOrCreateWebhook(
+    currentTextChannel,
+    'VAIT-AnimatedEmoji-Hook'
```

Comment:
should we use a generic name for our webhook? Individual webhook for each function is redundant.